docs/src/getting-started/installation.md

```diff
@@ -1,103 +1,310 @@
 
-### Installation via [asdf](https://asdf-vm.com/)
+```shell
+snforge --version
+```
 
-First, add the Starknet Foundry plugin to asdf:
+or
 
 ```shell
-$ asdf plugin add starknet-foundry
+sncast --version
 ```
 
-#### Common Error
+## Common Errors
+
+### No Version Set (Linux and MacOS Only)
 
 Users may encounter this error when trying to use `snforge` or `sncast` without setting a version:
 
 ```shell
 No version is set for command snforge
-Consider adding one of the following versions in your config file at starknet-foundry 0.32.0
+Consider adding one of the following versions in your config file at $HOME/.tool_versions
+starknet-foundry 0.32.0
 ```
 
 This error indicates that `Starknet Foundry` version is unset. To resolve it, set the version globally using asdf:
 
 ```shell
-$ asdf global starknet-foundry <version>
+asdf global starknet-foundry <version>
 ```
 
-For additional information on asdf version management, see the [asdf](https://asdf-vm.com/guide/getting-started.html#_6-set-a-version)
+For additional information on asdf version management, see
+the [asdf](https://asdf-vm.com/guide/getting-started.html#_6-set-a-version)
 
-### Installation on Windows
+### Invalid Rust Version
+
+When running any `snforge` command, error similar to this is displayed
+
+```shell
+Compiling snforge_scarb_plugin v0.34.0
+error: package snforge_scarb_plugin v0.34.0 cannot be built because it requires rustc 1.80.1 or newer, while the currently active rustc version is 1.76.0
+```
 
-As for now, Starknet Foundry on Windows needs manual installation, but necessary steps are kept to minimum:
+This indicates incorrect Rust version is installed or set.
 
-1. [Download the release](https://github.com/foundry-rs/starknet-foundry/releases) archive matching your CPU
-   architecture.
-2. Extract it to a location where you would like to have Starknet Foundry installed. A folder named snfoundry in
-   your [`%LOCALAPPDATA%\Programs`](https://learn.microsoft.com/en-us/windows/win32/shell/knownfolderid?redirectedfrom=MSDN#FOLDERID_UserProgramFiles)
-   directory will suffice:
+Verify if rust version >= 1.80.1 is installed
 
-```batch
-%LOCALAPPDATA%\Programs\snfoundry
+```shell
+rustc --version
+1.80.1
 ```
 
-3. Add path to the snfoundry\bin directory to your PATH environment variable.
-4. Verify installation by running the following command in new terminal session:
+To fix, follow the platform specific instructions:
 
-   ```shell
-   $ snforge --version
-   ```
+#### Linux and MacOS
 
-   and
+If the version is incorrect or the error persists, try changing the global version of Rust
 
-   ```
-   $ sncast --version
-   ```
+```shell
+rustup default stable
+```
 
-### Universal-Sierra-Compiler update
+and local version of Rust
+
+```shell
+rustup override set stable
+```
+
+#### Windows
+
+Follow [Rust installation](https://www.rust-lang.org/tools/install) and ensure correct version of rust was added to
+PATH.
+
+### `scarb test` Isn’t Running Tests Correctly
```

Comment:
What does `Running Tests Correctly` mean? Maybe let's rephrase it to e.g. `Scarb isn't using snforge` or `Scarb isn't running tests with snforge`